a4c-brooklyn-plugin/src/main/java/alien4cloud/brooklyn/BrooklynOrchestratorFactory.java

```diff
@@ -43,12 +46,14 @@ public void destroy(BrooklynOrchestrator instance) {
 
     @Override
     public Configuration getDefaultConfiguration() {
+        List<String> metadataProviders = MutableList.of(
+                BrooklynToscaTypeProvider.class.getName(),
+                DefaultToscaTypeProvider.class.getName());
```

Comment:
These are in the wrong order - DefaultToscaTypeProvider should be lowest priority i.e first in this list